gateway/enforcer/org.wso2.apk.enforcer.commons/src/main/java/org/wso2/apk/enforcer/commons/dto/JWTInfoDto.java

```diff
@@ -44,6 +44,7 @@ public class JWTInfoDto {
     private Map<String, String> appAttributes = new HashMap<>();
     private String sub;
     private String[] organizations;
+    private Map<String, String> claims = new HashMap<>();
```

Comment:
can we have String,Object
unless this convert int long values to string in jwt

Reply:
problem is how we're going to capture values from the CRs. And when comes to the protobuf, there's no such variable type called "Object". But we can use something like `Any` which again has a `ByteString` type. Can we extract the value from the CR and parse it to Int, Float or any other relevant primitive type if needed? @AmaliMatharaarachchi WDYT? Or we can let the values to be identified as `String` and allow the backends to do type conversions if needed